pandas/tests/test_common.py

```diff
@@ -154,8 +158,50 @@ def test_compression_size_fh(obj, method, compression_only):
         assert uncompressed > compressed
 
 
-# GH 21227
```

Comment:
can you move all of the compression tests to pandas/tests/io/test_common.py (including existing ones), may want to create a new class to organzie them

Reply:
Done in https://github.com/pandas-dev/pandas/pull/22011/commits/387d1d29f2833e236e0d8c3e3167c94614676973

Didn't create a new class as I didn't want to make the line-length a bigger issue than it already is.